CONTRIBUTING.md

```diff
@@ -142,5 +142,5 @@ To ensure smooth collaboration and to maintain a high-quality codebase, please a
 * `git log` can help you find the original/relevant authors of the code you are modifying. If you need, feel free to tag the author in your Pull Request comment if you need assistance or review.
 * Do not re-create a pull-request for the same change. Use one Pull Request related to the same change(s). The purpose here is to keep the history and all comments in the Pull Request.
 * Consider open questions and concerns in all comments of your Pull Request, provide replies and resolve addressed comments, if those don't serve reference purposes. If a comment doesn't contain `nit`, `minor`, or `not a blocker` mention, please provide feedback to the comment before merging.
-* Give time for review. For instance two working days is a good base to get first reviews and comments.
+* Give time for review. For instance two working days is a good base to get first reviews and comments for changes that are not likely to affect downstream projects. For changes that touch core interfaces and behaviours three working days would be advisable.
```

Comment:
We discussed what we consider integration points here : https://lists.apache.org/thread/0nj24zro7kyctqfnlml08ppo7zs9xcqs, wondering if we should link it here or be a bit precise on expectation on what interface we should give additional time to review

Reply:
It's true that some SPIs/APIs in Polaris are still not clearly defined. That said, we do have well-defined ones, for example, the REST Spec.

Also, I'd suggest that structural changes, such as adding or removing Java modules, should probably allow for a longer review window.

Reply:
Updated. PTAL.